### PR TITLE
Ansible playbook to role updates

### PR DESCRIPTION
**Pull request by carlosmmatos**
#### Description:

- fixed issue with when clause ordering
- minor change to readme template
- fix bug where the script adds commits even when there are no changes
  - This should only commit to the repo when a change is detected, not all the time as it currently stands.

### CODE REVIEW

File: utils/ansible_playbook_to_role.py [hunk 1]
```diff
@@ -396,9 +396,8 @@ def _local_content(self, filepath):
 
     def _remote_content(self, filepath):
         remote = self.remote_repo.get_contents(filepath)
-        content = remote.decoded_content
-        if filepath == 'README.md':
-            content = content.decode("utf-8")
+        # We rant the raw string to compare against _local_content
```
Comment (yuumasato) [inline utils/ansible_playbook_to_role.py:399]:
```suggestion
        # We want the raw string to compare against _local_content
```

**Review by yuumasato — APPROVED** [review]
LGTM, just found a small typo.

File: utils/ansible_playbook_to_role.py (hunk 1 above)
Comment (carlosmmatos) [reply utils/ansible_playbook_to_role.py:399]: lol whoops.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**